config/kubernetes/org.yaml

```diff
@@ -441,6 +445,7 @@ members:
 - idealhack
 - iheanyi1
 - ihmccreery
+- ike-ma
```

Comment:
@ehashman asked to wait till the first PR here: https://github.com/kubernetes/org/issues/2657 Please delay this membership assignment

Reply:
Thank you for pointing that out @SergeyKanzhelev. Updating the PR

Reply:
Updated the PR